fds/services/git_service.py

```diff
@@ -55,4 +55,25 @@ def add(self, add_argument: str) -> Any:
             execute_command(["git", "add", ignore_file])
 
     def commit(self, message: str) -> Any:
+        """
+        Commit to Git
+        :param message: Commit message
+        :return:
+        """
         execute_command(["git", "commit", "-am", message], capture_output=False)
+
+    def push(self, remote: str, ref: str) -> Any:
+        """
+        Push commits to Git remote
+        :return:
+        """
+        push_cmd = ["git", "push"]
+        if remote:
+            push_cmd.append(remote)
+            if ref:
+                push_cmd.append(ref)
+            else:
+                repo = pygit2.Repository(self.repo_path)
+                curr_branch = repo.head.shorthand
```

Comment:
WDYT about using `git rev-parse --abbrev-ref HEAD` instead of relying on an external lib which just does the same thing?
https://stackoverflow.com/a/6245587